### DISCUSSION TITLE
Is there any way to relate rows in multiple tables in budibase tables?

### DISCUSSION THREAD

**Discussion by shaikhashraf709** [Budibase/budibase#1134] [Help]
I am designing a large form for which i have multiple budibase tables such as personal_info, bank_details etc. and i have a screen for each table to breakdown the form into small pages and i don't find any way to refer a record in a table with another record in another table. (like primary and foreign key in sql).
i cant add a common column like "application id" in all the tables in order have a relation between tables as that column also shows up in all the screens in forms also the fields in a form are not editable.

basically i need a way to address an individual row in budibase table
Is there any way to relate rows in multiple tables in budibase tables?
